src/pt/sinensis/src/eu/kanade/tachiyomi/extension/pt/sinensis/SinensisScan.kt

```diff
@@ -1,56 +1,17 @@
 package eu.kanade.tachiyomi.extension.pt.sinensis
 
-import eu.kanade.tachiyomi.multisrc.madara.Madara
-import eu.kanade.tachiyomi.network.interceptor.rateLimit
-import eu.kanade.tachiyomi.source.model.SChapter
-import eu.kanade.tachiyomi.source.model.SManga
-import okhttp3.HttpUrl.Companion.toHttpUrl
-import okhttp3.OkHttpClient
-import org.jsoup.nodes.Element
+import eu.kanade.tachiyomi.multisrc.peachscan.PeachScan
 import java.text.SimpleDateFormat
 import java.util.Locale
-import java.util.concurrent.TimeUnit
 
-class SinensisScan : Madara(
+class SinensisScan : PeachScan(
     "Sinensis Scan",
-    "https://sinensisscan.net",
+    "https://sinensistoon.com",
     "pt-BR",
-    SimpleDateFormat("dd/MM/yyyy", Locale("pt", "BR")),
+    SimpleDateFormat("dd 'de' MMMM 'de' yyyy 'às' HH:mm", Locale("pt", "BR")),
 ) {
 
     // Name changed from Sinensis to Sinensis Scan
     override val id: Long = 3891513807564817914
```

Comment:
If theme changed, old URLs are likely no longer working. Since id for "Sinesis Scan" hasn't been used, it should be fine to just remove this line (instead of setting `versionId` to 2)

```suggestion
```